docs/README.md

```diff
@@ -48,6 +50,18 @@ make livehtml
 make.bat livehtml
 ```
 
+## Running Sphinx Locally with Docker
+
+To generate the documentation as HTML file, use the provided docker command from the project root:
```

Comment:
There should be an "an" here
```suggestion
To generate the documentation as an HTML file, use the provided docker command from the project root:
```

Reply:
It would also be nice if you could adapt the make text above as well, as there are two spaces between "To" and "generate" and there is also a missing "a".